clang/test/lab1/travin_maksim/test.cpp

```diff
@@ -0,0 +1,36 @@
+// RUN: %clang_cc1 -ast-dump -ast-dump-filter test -load %llvmshlibdir/TravinAlwaysInlinePlugin%pluginext -add-plugin add-always-inline %s | FileCheck %s
```

Comment:
Please add that no additional always_inline attribute is added if one is already present there?